### PR TITLE
Game of life implemented

### PR DESCRIPTION
**Pull request by KubaBBB**

### CODE REVIEW

File: GoF/game.py [hunk 1]
```diff
@@ -113,29 +183,32 @@ def main():
...
-	# set up animation 
+	logger = []
+	# set up animation
 	fig, ax = plt.subplots() 
 	img = ax.imshow(grid, interpolation='nearest') 
-	ani = animation.FuncAnimation(fig, update, fargs=(img, grid, N, ), 
-								frames = 10, 
+	ani = animation.FuncAnimation(fig, update, fargs=(img, grid, N, patternsGrid, logger),
+								frames = 10,
 								interval=updateInterval,
```
Comment (LemurPwned) [inline GoF/game.py]: Inconsitent naming patterns_grid, update_interval

File: GoF/game.py [hunk 2]
```diff
@@ -94,16 +155,25 @@ def main():
...
-	# set grid size 
+	parser.add_argument('--gosper', action='store_true', required=False)
+	parser.add_argument('--file', dest='file', required=False)
+
+	args = parser.parse_args()
+	#Trzeba robic reshape na glider bo statek moze sie poruszac w inne strony
+	patterns = ['block', 'tub', 'blinker_1', 'blinker_2', 'glider_1', 'glider_2', 'glider_3', 'glider_4', 'pond']
+	patternsGrid = dict()
+	for pattern in patterns:
+		patternsGrid[pattern] = read_pattern(pattern)
```
Comment (LemurPwned) [inline GoF/game.py]: same here, inconsistent naming

File: GoF/game.py (hunk 1 above)
Comment (KubaBBB) [reply GoF/game.py]: done

File: GoF/game.py (hunk 2 above)
Comment (KubaBBB) [reply GoF/game.py]: done

File: GoF/game.py [hunk 3]
```diff
@@ -3,30 +3,84 @@
...
+		content = f.readlines()
+	# you may also want to remove whitespace characters like `\n` at the end of each line
+	content = [x.strip() for x in content]
+	for index in range(content.__len__()):
+		sep = content[index].index(',')
+		values = [content[index][0:sep], content[index][sep+1:content[index].__len__()]]
+		grid[int(values[0])+i, int(values[1])+i] = 255
+
+def read_pattern(pattern):
+	with open(os.path.join(os.path.dirname(__file__), 'resources', 'patterns', f'{pattern}.txt')) as f:
```
Comment (LemurPwned) [inline GoF/game.py:27]: please use np.load instead of manually reading. Save the file as the numpy array and then read it back

**Review by LemurPwned — APPROVED** [review]

**Pull request merged**